microsoft-edge/devtools-guide-chromium/crash-analyzer/index.md

```diff
@@ -158,13 +158,13 @@ To use the **Crash analyzer** tool in DevTools:
 
    This loads the source maps that correspond to the source modules, and attempts to unminify any stack frames for which JavaScript or TypeScript source content was contained in the source map.
 
-1. If the source content was contained for a frame, click on the frame.  Click through the individual stack frames, to see the lines in your original code that caused the error.
+1. If the source code is available in your source maps for a frame, click on the frame to display the source code.
```

Comment:
according to product-level style guide overriding corp-level style guide, probably "click" not "click on".  might revise
https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui

Reply:
1. If the source code is available in your source maps for a frame, click the frame to display the source code.